Wrappers/Python/test/test_functions.py

```diff
@@ -876,7 +874,49 @@ def test_call_real_anisotropic(self):
         
         res1 = self.tv_aniso(x_real)
         res2 = self.grad.direct(x_real).pnorm(1).sum()
-        np.testing.assert_equal(res1, res2)                
+        np.testing.assert_equal(res1, res2)     
+
+    def test_strongly_convex_TV(self):
+
+        TV_no_strongly_convex = self.alpha * TotalVariation()
+        self.assertEqual(TV_no_strongly_convex.strong_convexity_constant, 0)
+
+        # TV as strongly convex, with "small" strongly convex constant
+        TV_strongly_convex = self.alpha * TotalVariation(strong_convexity_constant=1e-4)
+
+        # check call
+        x_real = self.ig_real.allocate('random', seed=4) 
+        res1 = TV_strongly_convex(x_real)
+        res2 = TV_no_strongly_convex(x_real) + (TV_strongly_convex.strong_convexity_constant/2)*x_real.squared_norm()
+        np.testing.assert_allclose(res1, res2, atol=1e-3)        
+
+        # check proximal
+        x_real = self.ig_real.allocate('random', seed=4) 
+        res1 = TV_no_strongly_convex.proximal(x_real, tau=1.0)
+        res2 = TV_strongly_convex.proximal(x_real, tau=1.0)  
+        np.testing.assert_allclose(res1.array, res2.array, atol=1e-3) 
+
+    @unittest.skipUnless(has_reg_toolkit, "Regularisation Toolkit not present")
+    def test_strongly_convex_CIL_FGP_TV(self):
+
+        FGP_TV_no_strongly_convex = self.alpha * FGP_TV()
+        self.assertEqual(FGP_TV_no_strongly_convex.strong_convexity_constant, 0)
+
+        # TV as strongly convex, with "small" strongly convex constant
+        FGP_TV_strongly_convex = self.alpha * FGP_TV(strong_convexity_constant=1e-3)
+
+        # check call
+        x_real = self.ig_real.allocate('random', seed=4) 
+        res1 = FGP_TV_strongly_convex(x_real)
+        res2 = FGP_TV_no_strongly_convex(x_real) + (FGP_TV_strongly_convex.strong_convexity_constant/2)*x_real.squared_norm()
+        np.testing.assert_allclose(res1, res2, atol=1e-3)        
+
+        # check proximal
+        x_real = self.ig_real.allocate('random', seed=4)         
+        res1 = FGP_TV_no_strongly_convex.proximal(x_real, tau=1.0)
+        res2 = FGP_TV_strongly_convex.proximal(x_real, tau=1.0)  
```

Comment:
I suppose here we would also want to check that `x_real` has remained with the same values. See comment above.

Reply:
Fixed